app/enterprise/0.36-x/deployment/installation/free-trial.md

```diff
@@ -0,0 +1,230 @@
+---
+title: How to Install a Kong Enterprise Free Trial with Docker
+---
+
+## Introduction
+
+This guide walks through downloading, installing, and starting a Kong Enterprise
+Free Trial using Docker. Your Free Trial license is active for 15 days. If you
+would like to extend your trial please contact your **Sales** associate.
+
+
+## Prerequisites
+
+To complete this guide you will need:
+
+- A valid [Kong Enterprise Free Trial](https://konghq.com/free-trial)
+- Docker
+- A Kong Enterprise Free Trial docker file (`.tar.gz`)
+- A valid Kong Enterprise Free Trial license file
+
+
+## Step 1. Load the Kong Enterprise Free Trial Docker Image
+
+1. Download the Kong Enterprise Trial Image file
+
+    The `kong-enterprise-edition-ft-0.36.tar.gz` file can be downloaded by
+    clicking the Docker link in the Kong Enterprise Free Trial Welcome email.
+    
+2. Load the Kong Enterprise Trial Image:
+
+    ```
+    $ docker image load --input kong-enterprise-edition-ft-0.36.tar.gz 
+    ```
+
+3. Run `docker images` to find the ID for the Kong Enterprise
```

Comment:
Would specify the repository name here in case they have many images running. 

```suggestion
3. Run `docker images` to find the ID for the `kong-docker-kong-free-trials-enterprise-edition-docker.bintray.io/kong-enterprise-edition`, the Kong Enterprise trial
```